tests/phpunit/CRM/Core/Payment/AuthorizeNetIPNTest.php

```diff
@@ -192,7 +192,7 @@ public function testIPNPaymentRecurSuccessSuppliedReceiveDate() {
     $this->assertTrue(substr($contribution['contribution_source'], 0, 20) == "Online Contribution:");
     $contributionRecur = $this->callAPISuccess('contribution_recur', 'getsingle', ['id' => $this->_contributionRecurID]);
     $this->assertEquals(5, $contributionRecur['contribution_status_id']);
-    $IPN = new CRM_Core_Payment_AuthorizeNetIPN(array_merge(['receive_date' => '1 July 2010'], $this->getRecurSubsequentTransaction()));
+    $IPN = new CRM_Core_Payment_AuthorizeNetIPN(array_merge(['receive_date' => '2010-07-01'], $this->getRecurSubsequentTransaction()));
```

Comment:
test failure was related to the improper value sent for the receive date. Looks like we should always provide an iso or the MySQL format?